docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -51,6 +51,10 @@ You want to predict the price value, which is a real value, based on the other f
 
     In **Solution Explorer**, right-click the project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the **Browse** tab, search for **Microsoft.ML**, select that package in the list, and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed.
 
+1. Install the **Microsoft.Data.DataView** Nuget Package:
```

Comment:
@luisquintanilla in which cases is it really necessary? When I updated the clustering tutorial (dotnet/samples#625), a mere updated of the `Microsoft.ML` package to v0.10 was enough. The DataView package gets installed automatically as `Microsoft.ML` depends on it.

Reply:
@pkulikov thanks for catching this. I have since removed the install instructions.